worlds/hk/__init__.py

```diff
@@ -219,8 +220,10 @@ def create_items(self):
                     if location_name == "Start":
                         self.world.push_precollected(item)
                     else:
-                        self.create_location(location_name)
                         pool.append(item)
+                        self.create_location(location_name).progress_type = LocationProgressType.PRIORITY if \
+                        option_choice == 2 else LocationProgressType.EXCLUDED if option_choice == 3 else \
+                        LocationProgressType.DEFAULT
```

Comment:
I feel like this might be cleaner as a dictionary lookup or similar, or could stand to be formatted nicer.

Reply:
done